mula/scheduler/context/context.py

```diff
@@ -30,9 +31,6 @@ def __init__(self) -> None:
         """Initializer of the AppContext class."""
         self.config: settings.Settings = settings.Settings()
 
-        if not self.config.database_dsn.startswith("postgresql"):
```

Comment:
I think this accidentally reverse a change.

Reply:
Initially, it was removed accidentally, however, I've moved it a bit lower on line 87